README.md

```diff
@@ -131,7 +131,12 @@ If you cannot find suitable test data on this repository, please contact us on t
       * test{,2}.genome.vcf.idx: Index feature file based on `test{,2}.genome.vcf` file
     * vcf:
       * test.rnaseq.vcf: RNAseq vcf corresponding to `test.rnaseq_{1,2}` reads
-
+    * yak:
+      * test.yak: Yak kmer index of 1000 of paternal paired-end reads from the GIAB Ashkenazim trio [RM8392](https://www-s.nist.gov/srmors/view_detail.cfm?srm=8392). These reads were selected from D2_S1_L001_R{1,2}_001.fastq.gz and D2_S1_L001_R{1,2}_002.fastq.gz so that they map to `pacbio/fastq/test_hifi.fastq.gz`.
+      * test2.yak: Yak kmer index of 1000 of maternal reads from the GIAB Ashkenazim trio [RM8392](https://www-s.nist.gov/srmors/view_detail.cfm?srm=8392). These reads were selected from D3_S1_L001_R{1,2}_001.fastq.gz and D3_S1_L001_R{1,2}_001.fastq.gz so that they map to `pacbio/fastq/test_hifi.fastq.gz`.
+  * pacbio:
```

Comment:
I can't edit this file otherwise I would!! 😅 

We need a separate section for pacbio in the README. Just need to dedent I think and add a new line.

Reply:
But this `pacbio` is just a platform like Illumina, so probably it should be within each species? Currently it looks fine to me:
```
homo_sapience
  genome
  illumina
  pacbio
```
  ?